prometheus_client/gc_collector.py

```diff
@@ -5,15 +5,21 @@
 import gc
 import os
 import time
+from collections import defaultdict
 
-from .metrics import Histogram
+from .metrics_core import HistogramMetricFamily
 from .registry import REGISTRY
+from .utils import INF
 
 
 class GCCollector(object):
     """Collector for Garbage collection statistics."""
 
-    def __init__(self, registry=REGISTRY, gc=gc):
+    _LATENCY = 'latency'
```

Comment:
I assume this is needed to avoid allocs?

Reply:
No it is just type error protection .